manim/mobject/geometry.py

```diff
@@ -3015,6 +3019,62 @@ def __init__(
 
         self.set_points(angle_mobject.get_points())
 
+    def get_lines(self) -> VGroup:
+        """Get the lines forming an angle of the :class:`Angle` class.
+
+        Returns
+        -------
+        :class:`~.VGroup`
+            A :class:`~.VGroup` containing the lines that form the angle of the :class:`Angle` class.
+
+        Examples
+        --------
+        ::
+
+            >>> line_1, line_2 = Line(ORIGIN, RIGHT), Line(ORIGIN, UR)
+            >>> angle = Angle(line_1, line_2)
+            >>> angle.get_lines()
+            VGroup(Line, Line)
+        """
+
+        return VGroup(self.line1, self.line2)
+
+    def get_value(self, degrees: bool = False) -> float:
+        """Get the value of an angle of the :class:`Angle` class.
+
+        Parameters
+        ----------
+        degrees
+            A boolean to decide the unit (deg/rad) in which the value of the angle is returned.
+
+        Returns
+        -------
+        :class:`float`
+            The value in degrees/radians of an angle of the :class:`Angle` class.
+
+        Examples
+        --------
+
+        .. manim:: GetValueExample
+            :save_last_frame:
+
+            class GetValueExample(Scene):
+                def construct(self):
+                    line1 = Line(LEFT+(1/3)*UP, RIGHT+(1/3)*DOWN)
+                    line2 = Line(DOWN+(1/3)*RIGHT, UP+(1/3)*LEFT)
+
+                    angle = Angle(line1, line2, radius=0.4)
+
+                    value = DecimalNumber(angle.get_value(degrees=True), unit="^{\\circ}")
+                    value.move_to(Angle(line1, line2, radius = 0.4 + 3*SMALL_BUFF).point_from_proportion(0.5))
```

Comment:
```suggestion
                    value.next_to(angle, UR, buff=0)
```
As seen here: https://manimce--1881.org.readthedocs.build/en/1881/reference/manim.mobject.geometry.Angle.html#manim.mobject.geometry.Angle , the number crosses with the angle which makes for an anesthetic look. I still feel the much easier version above works and looks better.